package.json

```diff
@@ -107,7 +107,10 @@
                     },
                     "markdownDescription": "%settings.interpreter.description%",
                     "scope": "resource",
```

Comment:
would you mind removing this too please? 😊

Reply:
Sure, done @luabud. Thank you!